core/src/main/java/io/questdb/griffin/engine/functions/groupby/MinCharGroupByFunction.java

```diff
@@ -77,10 +77,29 @@ public int getValueIndex() {
     }
 
     @Override
-    public boolean isParallelismSupported() {
+    public boolean isConstant() {
         return false;
     }
 
+    @Override
+    public boolean isParallelismSupported() {
+        return UnaryFunction.super.isParallelismSupported();
+    }
+
+    @Override
+    public boolean isReadThreadSafe() {
+        return UnaryFunction.super.isReadThreadSafe();
+    }
+
+    @Override
+    public void merge(MapValue destValue, MapValue srcValue) {
+        char srcMin = srcValue.getChar(valueIndex);
+        char destMin = destValue.getChar(valueIndex);
```

Comment:
nit: MapValue has functions such as `addInt()` or similar, which can add value in single map lookup. Similarly we could have `MapValue.setMin(index, value)`

Reply:
We don't do any map lookups in this method, only value lookups. Value lookups are relatively cheap since we pre-calculate all offsets.

Reply:
Added maxInt, minInt and minLong in 19e0a874b16a100e7181e4bc9bfb089e733b8c4e and 1972c4394897612cf0740dd1cd8acd0113048509